src/time_series_formats.jl

```diff
@@ -25,6 +25,33 @@ function read_time_series(file_path::AbstractString, component_name = nothing; k
     return read_time_series(get_time_series_format(file), file, component_name; kwargs...)
 end
 
+"""
```

Comment:
Should we make our own format in JSON?  We could include other information like resolution and component name/type.  Or are you thinking that a separate `timeseries_pointers.json` would have that information and point to this file?

Reply:
I think using the same pointers files with extension is the way to go.

Reply:
My preference is one JSON file that contains all relevant data and metadata.  It would be very clear to users on what they need to provide.  I think the reason that we have the timeseries_pointers.json combined with raw CSV is that we are working with existing file formats.  If the same thing is true for this type of forecast data then I'm certainly OK staying with it.  However, if the user is going to have to modify a bunch of things anyway, I would prefer to go with one self-contained file that is extensible.